cmd/dp-import-api/main.go

```diff
@@ -54,23 +57,75 @@ func main() {
 	}
 
 	router := mux.NewRouter()
-	s := server.New(config.BindAddr, router)
-	log.Debug("listening...", log.Data{
-		"bind_address": config.BindAddr,
-	})
+	router.Path("/healthcheck").HandlerFunc(healthcheck.Handler)
+
+	httpServer := server.New(config.BindAddr, router)
+	httpServer.HandleOSSignals = false
+
+	signals := make(chan os.Signal, 1)
+	signal.Notify(signals, os.Interrupt, os.Kill, syscall.SIGTERM)
```

Comment:
Do we need to listen to `os.Kill`? As I thought that it can not be caught by a process?

Reply:
correct - well spotted - removed